rewriter/src/main/java/com/facebook/coresql/rewriter/OrderByRewriter.java

```diff
@@ -35,31 +36,38 @@
 public class OrderByRewriter
         extends Rewriter
 {
-    PatternMatcher<Set<AstNode>> matcher;
     public Set<AstNode> patternMatchedNodes;
     private static final String REWRITE_NAME = "ORDER BY without LIMIT";
 
     public OrderByRewriter()
     {
-        this.matcher = new OrderByPatternMatcher();
         this.patternMatchedNodes = new HashSet<>();
     }
 
     @Override
     public boolean rewritePatternIsPresent(String sql)
```

Comment:
Generally for boolean methods they are either named `isXxx` or `hasXxx` so here it probably should just be named `hasPattern` or `hasRewritePattern`

Reply:
Done